src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,30 +1,58 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import React from 'react';
+import { SortType } from '../../types/SortType';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  query: string;
+  onQueryChange: (query: string) => void;
+  onSelect: (query: SortType) => void;
+  onClear: () => void;
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button data-cy="clearSearchButton" type="button" className="delete" />
-      </span>
-    </p>
-  </form>
-);
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  onQueryChange,
+  onSelect,
+  onClear,
+}) => {
+  return (
+    <form className="field has-addons">
+      <p className="control">
+        <span className="select">
+          <select
+            data-cy="statusSelect"
+            onChange={event => onSelect(event.target.value as SortType)}
+          >
+            <option value="all">{SortType.ALL}</option>
+            <option value="active">{SortType.ACTIVE}</option>
+            <option value="completed">{SortType.COMPLETED}</option>
```

Comment:
```suggestion
            <option value={SortType.ALL}>{SortType.ALL}</option>
            <option value={SortType.ACTIVE}>{SortType.ACTIVE}</option>
            <option value={SortType.COMPLETED}>{SortType.COMPLETED}</option>
```

But even better try to use `map` here